### ISSUE TITLE
Invalid Start Time

### ISSUE THREAD

**Issue by Usman264** [BeanBagKing/EventFinder2#6]
When Ever I click the FInd Events button it give "Invalid Start Time" although I am sure about the starting and ending time ..

**Comment by shaneholloman** [comment]
Same issue here @Usman264

**Comment by BeanBagKing** [comment]
I don't think I've ever seen "Invalid Start Time". If I malform the date/time, I can get "String was not recognized as a valid DateTime.". Can you post some screenshots, of both the date/time string and the error? A copy and paste of the error text (if applicable). etc.?

My only initial though is that it might be localization. My poorly coded app is expecting a start time in one format, but your system is providing it in another. Do you mind providing your country if it's other than US, or how your system presents the date/time format?

**Comment by shaneholloman** [comment]
@BeanBagKing 

You are correct. Start and end times on my system are formatted differently from what the app initially suggests
`08/25/2021 02:35:24` <= from your app, hitting "Start Time". 
`25/08/2021 02:35:24` <= My system expects

This why some folks get invalid time messages

![png](https://i.imgur.com/NWRkV2e.png)
